src/content/en/tools/lighthouse/audits/tap-targets.md

```diff
@@ -0,0 +1,68 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: How to pass the "Tap targets are not sized appropriately" Lighthouse audit.
+
+{# wf_updated_on: 2019-02-15 #}
+{# wf_published_on: 2019-02-15 #}
+{# wf_blink_components: Platform>DevTools #}
+
+# Tap targets are not sized appropriately {: .page-title }
+
+## Overview {: #overview }
+
+[webmasters]: https://webmasters.googleblog.com/2015/04/rolling-out-mobile-friendly-update.html
+[MDN]: https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect#Value
+
+Tap targets are interactive elements, like buttons or links, that users frequently tap.
+Appropriately-sized tap targets make pages more mobile-friendly and accessible. Google Search
+started boosting the ranking of mobile-friendly pages on mobile search results back in 2015.
+See [Rolling out the mobile-friendly update][webmasters].
+
+A tap target is inappropriately-sized when its bounding rectangle is less than 48 pixels. 48
+pixels is about the size of a human finger. Any target smaller than a finger requires a
+lot of precision to tap. The bounding rectangle is the target's `width` (or `height`) +
+`padding` + `border` + `margin`. Both of the icons in **Figure 2** are appropriately-sized.
+
+<figure>
+  <img src="/web/fundamentals/accessibility/imgs/touch-target.jpg"
```

Comment:
Is this image in the PR?

Reply:
Make sure the examples actually fail in LH, since sometimes we try to be lenient 😂

Reply:
The image already exists in the repo